packages/site/pages/components/datawell.js

```diff
@@ -92,31 +93,67 @@ export default withServerProps(_ => (
         ]}
       />
 
-      <SectionHeading>In-app example</SectionHeading>
+      <SectionHeading>Example</SectionHeading>
```

Comment:
Usually these headings are specific to what you're showing exercised, often a prop.  Is it labels that you're highlighting here?  I'd probably go for "Labels" here instead.